### PR TITLE
Suez Water sensor documentation

### PR DESCRIPTION
**Pull request by ooii**
**Description:**
This is documentation for the new Suez Water sensor. 
It describes how the sensor should be used and what information it provides.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23844

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/suez.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: suez.png
+ha_release: 0.95
+ha_category:
+  - Water
+ha_iot_class: Cloud Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/suez.markdown]: Please remove the `redirect_from`

File: source/_components/suez.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+title: "Suez Water Sensor"
+description: "Instructions on how to integrate Suez Water daily data within Home Assistant."
+date: 2019-06-06 11:11
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: suez.png
+ha_release: 0.95
+ha_category:
```
Comment (klaasnicolaas) [inline source/_components/suez.markdown:6]:
```suggestion
ha_category:
  - Water
  - Sensor
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/suez.markdown (hunk 1 above)
Comment (ooii) [reply source/_components/suez.markdown]: Done

File: source/_components/suez.markdown [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  - Water
```
Comment (fabaff) [inline source/_components/suez.markdown]: Remove this. It would create an additional category with one entry.

Comment (ooii) [reply source/_components/suez.markdown]: Done

**Review by frenck — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
✅ Can be merged after merging the parent PR 👍

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @ooii! Good job! 👍

**Pull request merged**